plugins/org.python.pydev.parser/src/org/python/pydev/parser/prettyprinterv2/IPrettyPrinterPrefs.java

```diff
@@ -34,6 +34,8 @@
 
     String getAssignPunctuation();
 
+    String getAssignPunctuationColon();
```

Comment:
I think that this could be named `getTypePunctuationColon()` as typing can be set on other places (not just in assigns).